lesson_15/tdd/tdd_app/src/test/java/com/codedifferently/lesson15/EmployeeManagertest.java

```diff
@@ -0,0 +1,49 @@
+package com.codedifferently.lesson15;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class EmployeeManagertest {
+  private EmployeeManager employeeManager;
+  private Map<Integer, Employee> employees;
```

Comment:
This doesn't make sense to me. How does this help you test the `EmployeeManager` functionality?